src/Evolution/Systems/CurvedScalarWave/Equations.hpp

```diff
@@ -21,8 +21,10 @@ struct Psi;
 struct Pi;
 template <size_t Dim>
 struct Phi;
+namespace Tags {
 struct ConstraintGamma1;
 struct ConstraintGamma2;
+}  // namespace Tags
 }  // namespace CurvedScalarWave
 /// \endcond
 
```

Comment:
In the help text below (and throughout the CurvedScalarWave system), please update to current naming conventions: \alpha, \beta^i, \gamma_{ij} for lapse, shift, spatial metric, etc.

Reply:
changed everywhere (I think !)